docs/index.html

```diff
@@ -705,17 +712,16 @@ <h1 id='api-usage-and-access-guidelines'>API usage and access guidelines</h1>
 
 <p>Without CA certificate, curl fails to verify that HTTP connections are secure and curl commands may fail with the SSL
 certificate problem. Provide the root CA certificate to curl for secure SSL communication.</p>
-<pre class="highlight plaintext"><code>curl -v --cacert {path}/rootCA.crt 'https://{odimra_host}:{port}/redfish/v1'
-</code></pre>
+
 <p><strong>1.</strong> If you are running curl commands on the server where the resource aggregator is deployed, provide the
     <code>rootCA.crt</code> file as shown in the curl command:</p>
-
+<pre class="highlight plaintext"><code>curl -v --cacert {path}/rootCA.crt 'https://{odimra_host}:{port}/redfish/v1'
+</code></pre>
 <p>{path} is where you have generated certificates during the deployment of the resource aggregator.</p>
 
 <p><strong>2.</strong> If you are running curl commands on a different server, perform the following steps to provide the rootCA.crt file.</p>
 
-<p>a. Navigate to <code>~/ODIM_v1.0/configuration/Odim/certificates</code> on the server where the resource
-         aggregator is deployed.</p>
+<p>a. Navigate to <code>~/ODIM_v1.0/configuration/Odim/certificates</code> on the server where the resource aggregator is deployed.</p>
 

```

Comment:
Can you check if there is version with ODIM folder?
that is,  ODIM_v1.0 or it should be just ODIM?